charts/linkerd2-cni/README.md

```diff
@@ -13,7 +13,7 @@ The following table lists the configurable parameters of the Linkerd2-cni chart
 
 | Parameter                            | Description                                                           | Default                       |
 |--------------------------------------|-----------------------------------------------------------------------|-------------------------------|
-`cniPluginImage`                      | Docker image for the CNI plugin                                       |`gcr.io/linkerd-io/cni-plugin`|
```

Comment:
It looks like master probably changed under you since you branched. Can you merge master into your branch when you get a chance to avoid these unrelated changes?